### PR TITLE
DOCS-7560 CI Visibility Explorer + Search Test Run or Pipeline Execution Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Update descriptions for test, pipeline, and common facets + improves linking from the Docs Glossary and doc pages.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by drichards-87** [comment]
Created DOCS-8065 for editorial review.

**Review by aliciascott — APPROVED** [review]
looks great!

File: content/en/continuous_integration/explorer/facets.md [hunk 1]
```diff
@@ -17,7 +17,52 @@ Facets are user-defined tags and attributes from your tests or pipelines. They a
...
+| Pipeline Name | Name of the CI pipeline. |
+| Node Name | Name of the CI node that executed the pipeline, stage, or job. |
+| Node Labels | Labels associated with the CI node that executed the pipeline, stage, or job. |
+| Pipeline URL | Provider URL for a pipeline execution. |
+| Pipeline ID | ID of the pipeline. |
+| Pipeline Number | Execution number of a CI pipeline, provided by the CI Provider. This increases when partially retrying a pipeline. |
+| Job URL | Provider URL for a job execution. |
+| Stage Name | Name of the CI stage. |
+| Job Name | Name of the CI job. |
+| Pod Name | Kubernetes Pod Name. |
```
Comment (ManuelPalenzuelaDD) [inline content/en/continuous_integration/explorer/facets.md:36]:
```suggestion
```
We ended up removing the description of the pod name facet

**Review by ManuelPalenzuelaDD — APPROVED** [review]
LGTM, thank you for the change! (left a small comment)

**Pull request merged**